src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -23085,5 +23085,35 @@ .maxstack  3
 }
 ");
         }
+
+        [Fact]
+        [WorkItem(18738, "https://github.com/dotnet/roslyn/issues/18738")]
+        public void Test()
+        {
+            var source = @"
+class C
+{
+    void M()
+    {
+        int? e = 5;
+        (int, string) y = (e, null);
```

Comment:
Is the fact that this sample has no implicit conversion, but does have an explicit one relevant?  The PR description doesn't say so.  If it isn't relevant should we add tests for similar cases where there is no implicit or explicit conversion?